draco/analysis/beamform.py

```diff
@@ -0,0 +1,515 @@
+import numpy as np
+from caput import mpiarray, config, mpiutil, pipeline
+from ..core import task, containers, io
+from cora.util import units
+from ch_util import tools, ephemeris
```

Comment:
You can't depend on `ch_util` in `draco`.

Reply:
I am not sure what the best solution is here.

I can move `polarization_map()` to ch_pipeline.util.tools.py (which I would create).
But then I would depend on ch_pipeline, is that ok?

Reply:
No. The point is draco can't depend on private CHIME packages.

I think you can mostly decode this by looking at the telescope `_beamclass` attribute. I think it's 0 for X pol and 1 for Y pol.